src/julia/pyjulia_helper.jl

```diff
@@ -44,6 +48,53 @@ if VERSION >= v"0.7-"
     end
 end
 
+
+macro prepare_for_pyjulia_call(ex)
+    
+    # f(x) should return a transformed expression x and whether to recurse 
+    # into the new expression
+    function stoppable_walk(f, x)
+        (fx, recurse) = f(x)
+        walk(fx, (recurse ? (x -> stoppable_walk(f,x)) : identity), identity)
+    end
+    
+    locals = gensym("locals")
+    globals = gensym("globals")
+    
+    function make_pyeval(expr, options...)
```

Comment:
```suggestion
    function make_pyeval(expr::Union{String,Symbol}, options...)
```

seems to be safer as we are doing `code = string(expr)`.